src/cli/main.c

```diff
@@ -108,16 +119,18 @@ main(int argc, char *argv[])
 			network_name = new_name;
 		} 
 
+		// TODO(sneha): is this where we should pass the version variable? 
 		if (ndb_provisioning(provcode, network_name) < 0)
 			usage();
 		else
 			goto out;
 
-	} else if (network_name)
+	} else if (network_name) {
+		// TODO(sneha): Is this where we should pass the version variable? 
```

Comment:
Here is where we build the json with the agent information sent to the controller: https://github.com/netvfy/netvfy-agent/blob/master/src/control.c#L186